dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,64 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseInspectorOptions):
+
+    def __init__(self, is_enabled=True, is_case_sensitive=True,
+                 stop_words=None):
+        """
+        Constructs the TextProfilerOption object with default values.
+
+        :ivar is_enabled: boolean option to enable/disable the option.
+        :vartype is_enabled: bool
+        :ivar is_case_sensitive: option set for case sensitivity.
+        :vartype is_case_sensitive: bool
+        :ivar stop_words: option set for stop words.
+        :vartype stop_words: Union[None, list(str)]
+        :ivar words: option set for word update.
+        :vartype words: BooleanOption
+        :ivar vocab: option set for vocab update.
+        :vartype vocab: BooleanOption
+        """
+        self.is_case_sensitive = is_case_sensitive
+        self.stop_words = stop_words
+        self.words = BooleanOption(is_enabled=True)
+        self.vocab = BooleanOption(is_enabled=True)
+        super().__init__(is_enabled=is_enabled)
```

Comment:
not gong to prevent on this but, but i would suggest putting this as the first call. There are other cases we have in which it is needed to call after, but when this is last, we open the option for something in the base's construction to override the the init. This could ultimately be dangerous for this case specifically, but not guaranteed to happen.
e.g. if a parent class decided to add `self.stop_words = False` in its `__init__` it would override the setting here since it is called last while one would assume that the child class's props in the `init` would be the final to be set.

Reply:
not sure if it has any effect now, but moved to the first call to prevent potential issues